packages/mgt-components/src/components/mgt-person/mgt-person.ts

```diff
@@ -853,6 +875,11 @@ export class MgtPerson extends MgtTemplatedComponent {
     };
     userPresenceClass[statusClass] = true;
 
+    const userPresenceClasses = {
+      'user-presence': true,
+      'user-presence__people': this._usage === 'people'
+    };
+
     return html`
       <div class=${classMap(userPresenceClass)} title=${presence.activity} aria-label=${presence.activity} role="img">
```

Comment:
Does this classMap need to also emit the user-presence__people class too? 
Perhaps that property needs to be added to the userPresenceClass object and the userPresenceClasses object can be removed?

Reply:
Yes, I agree that it's unused in this case and should be removed. I'm making this update.